MetaMask/metamask-extension
# Improve confirmation screens for account management snaps

## **Description**

Redesigns the confirmation screens used to manage account management snaps according to the Figma. Also bumps ApprovalController to the latest version to use new features that allow for more customization of success/failure screens. Also fixes some alignment issues and styling problems on the snap redirection screen.

## **Related issues**

Closes https://github.com/MetaMask/MetaMask-planning/issues/1765

## **Manual testing steps**

1. Go to https://metamask.github.io/snap-simple-keyring/latest/
2. Connect to the snap
3. Create an account
4. Remove the same account
5. Notice that the flows have changed

Additionally, sign using the redirection flow and notice that the screen is properly aligned.

## **Screenshots/Recordings**




https://github.com/MetaMask/metamask-extension/assets/1561200/eee89ee8-40e2-4ff4-87c0-a0ef6db1c697
